lib/game_teams_collection.rb

```diff
@@ -13,4 +13,12 @@ def create_game_teams_array(file_path)
 
     csv.map { |row| GameTeams.new(row) }
   end
+
+  def game_teams_hash
```

Comment:
can be added as a instance variable to increase performance

Reply:
yes!